src/radial-chart/index.js

```diff
@@ -235,19 +238,64 @@ class RadialChart extends React.Component {
             className="rv-radial-chart__series--pie"
             transform={`translate(${width / 2},${height / 2})`}
             ref="container">
-            {data.map((d, i) =>
-              <path {...{
-                d: arc(pieData[i]),
-                style: {
-                  opacity: opacityFunctor && opacityFunctor(d),
-                  stroke: strokeFunctor && strokeFunctor(d),
-                  fill: fillFunctor && fillFunctor(d)
-                },
-                onMouseEnter: e => this._sectionHandler(onSectionMouseOver, pieData[i], e),
-                onMouseLeave: e => this._sectionHandler(onSectionMouseOut, pieData[i], e),
-                onClick: e => this._sectionHandler(onSectionClick, pieData[i], e),
-                key: i
-              }}/>)}
+            <g className="rv-radial-chart__series--pie-slices-wrapper">
+              {data.map((d, i) =>
+                <path {...{
+                  className: 'rv-radial-chart__series--pie__slice',
+                  d: arc(pieData[i]),
+                  style: {
+                    opacity: opacityFunctor && opacityFunctor(d),
+                    stroke: strokeFunctor && strokeFunctor(d),
+                    fill: fillFunctor && fillFunctor(d)
+                  },
+                  key: i
+                }}/>)}
+            </g>
+            <g className="rv-radial-chart__series--pie-labels-wrapper">
+              {showLabels && data.map((d, i) => {
+                // reject the label if its not affixed to the section
+                const canRenderMainLabel = d.label && typeof d.label === 'string';
+                const canRenderSubLabel = d.subLabel && typeof d.subLabel === 'string';
+                if (!canRenderMainLabel && !canRenderSubLabel) {
+                  return;
+                }
+                const angle = (pieData[i].startAngle + pieData[i].endAngle) / 2 - Math.PI / 2;
```

Comment:
It would be better
const angle = (pieData[i].startAngle + pieData[i].endAngle) / 2 - (Math.PI / 2);
also explain what does this equation do.